environments/action-hooks/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "private": true,
+  "name": "action-hooks",
+  "version": "0.0.1",
+  "scripts": {
+    "pull": "amplify pull --appId d1r9l08y32q5fb --envName dev"
+  }
+}
```

Comment:
Perhaps in a future PR we can add a readme file for this environment with what attributes are enabled, that look similar to the ones we have already for other environments.

Reply:
Pushed up a README

Reply:
👍